h2/src/tools/org/h2/build/Build.java

```diff
@@ -668,7 +668,9 @@ public void javadocImpl() {
                 File.pathSeparator + "ext/lucene-queryparser-5.5.5.jar" +
                 File.pathSeparator + "ext/org.osgi.core-4.2.0.jar" +
                 File.pathSeparator + "ext/org.osgi.enterprise-4.2.0.jar" +
-                File.pathSeparator + "ext/jts-core-1.15.0.jar",
+                File.pathSeparator + "ext/jts-core-1.15.0.jar" +
+                File.pathSeparator + "ext/asm-6.1.jar" +
+                File.pathSeparator + "ext/junit-4.12.jar",
```

Comment:
`./build.sh docs` completes normally without these imports. Why you need them?

The same below.

Reply:
It is necessary for `./build.sh javadocImpl`
This step checks and builds javadoc for tests. 
Junit4 and asm were added to tests code.

Reply:
Cleaned up for `javadocImpl3`. Tests dir isn't included there.